src/test/java/ru/training/at/hw2/ex1/ExerciseFirst.java

```diff
@@ -0,0 +1,177 @@
+package ru.training.at.hw2.ex1;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import ru.training.at.hw2.TestBaseClass;
+
+public class ExerciseFirst extends TestBaseClass {
+
+    SoftAssert softAssert = new SoftAssert();
+
+    @BeforeTest
+    @Override
+    public void beforeTest() {
+        super.beforeTest();
+    }
```

Comment:
What is the purpose of the overriding this method here?

Reply:
I wanted to call parent's before method by this way. It was my night "right" thought))
But I have checked that parent's before and after methods would execute in any way. So overriding is unnecessary here.